Replicator/Replicator.cc

```diff
@@ -1307,4 +1295,36 @@ namespace litecore { namespace repl {
         }
     }
 
+    void Replicator::setMsgHandlerFor3_0_Client(Retained<blip::MessageIn> request)
```

Comment:
This method takes a while to understand and so deserves some comments.  It is a no-op for active replicators, messages that include an explicit collection property, and the second time and after that this method is called.  The middle part is particularly important to note because that is what stops every 3.1 based message from causing trouble here.